app/controllers/application_controller.rb

```diff
@@ -29,7 +29,15 @@ def redirect_after_login
   def redirect_forward
     path = session[:forward_to]
     session[:forward_to] = nil
-    redirect_to path
+    if path == "orders_path"
+      redirect_to seller_orders_path(current_user.slug)
+    elsif path == "seller_store"
+      redirect_to seller_store_path(current_user.slug)
+    elsif path == "seller_dashboard"
+      redirect_to seller_dashboard_path(current_user.slug)
+    else
+      redirect_to path
+    end
```

Comment:
not the prettiest, but it works. I think anoter option would be to have a hash and every forward_to passes in a key and then you find the appropriate path from the hash values. This will work for now for time constraints